### DISCUSSION TITLE
Notifications have a white padding

### DISCUSSION THREAD

**Discussion by showierdata9978** [community/community#113379] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/68120127/73697be1-5623-485e-b780-40b235dbea81)

**Comment by santoshvandari** [comment]
May be it's a browser render problem. Did you try with another browser? I didn't find any white padding on my device.

**Comment by showierdata9978** [comment, marked as answer]
Its gone now, don't know why it was there, i have not done any ext changes since then

**Comment by calenzo** [comment]
lol
